src/components/AllCitiesOverviewContainer/AllCitiesOverview/AllCitiesOverview.js

```diff
@@ -82,24 +82,47 @@ const AllCitiesOverview = props => (
       </div>
     </div>
 
-    <div className={style.subInfoContainer}>
-      <div className={style.subInfoTextWrapper}>
-        <h1 className={style.infoTitle}>Policy priorities & context</h1>
+    <div className={style.subInfoWrapper}>
+      <div className={style.subInfoContainer}>
+        <h1 className={style.infoTitle}>
+          <span className={style.infoTitleHeavy}>Policy priorities</span> & Context
+        </h1>
 
         <p className={style.subInfoBodyText}>
           There are 6 policy priorities to help achieve
           the objectives of the {smartCitiesLink}. The National Cities Performance
           Framework is designed to measure how well our cities are performing, using
-          two types of indicators:
+          two types of indicators.
         </p>
-      </div>
 
-      <Icon
-        className={style.subInfoIcon}
-        icon="city"
-        size={120}
-        color={COLORS.PRIMARY_600}
-      />
+        <dl className={style.legend}>
+          <dt className={style.term}>
```

Comment:
I feel this would be cleaner and simpler CSS if you wrapped each row in a `<div>`

Reply:
Indeed! I was under the impression it wasn't valid to do that, but appears it is!

Reply:
I had to Google it first, but yep, valid.